assets/src/edit-story/components/panels/textStyle/color.js

```diff
@@ -27,15 +28,32 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import { Color, Label, Row } from '../../form';
-import { useCommonColorValue } from '../utils';
+import { Color, Label, Row, ToggleButton } from '../../form';
+import { useCommonColorValue, getCommonValue } from '../utils';
+import { BACKGROUND_TEXT_MODE } from '../../../constants';
+import { ReactComponent as FilledIcon } from '../../../icons/fill_filled_icon.svg';
+import { ReactComponent as HighlightedIcon } from '../../../icons/fill_highlighted_icon.svg';
+
+const FillRow = styled(Row)`
+  align-items: flex-start;
+  justify-content: flex-start;
+`;
+
+const FillLabel = styled(Label)`
+  flex-basis: 45px;
+`;
+
+const Space = styled.div`
+  flex: ${({ flex }) => flex};
+`;
 
 function ColorControls({ selectedElements, pushUpdate }) {
   const color = useCommonColorValue(selectedElements, 'color');
   const backgroundColor = useCommonColorValue(
     selectedElements,
     'backgroundColor'
   );
+  const backgroundType = getCommonValue(selectedElements, 'backgroundType');
```

Comment:
Consistently with the `BACKGROUND_TEXT_MODE`, let's call this property `backgroundTextMode`. Also please add it to the `types.js` for text elements.